tools/csv-generator/csv-generator.go

```diff
@@ -159,6 +161,33 @@ Performance Addon Operator provides the ability to enable advanced node performa
 		operatorCSV.Annotations["olm.skipRange"] = *skipRange
 	}
 
+	// Setup the Conversion Webhook
```

Comment:
we unconditionally add the following. Which is OK. But question is if we can have directly in the CSV template, and if not why.

Reply:
The question is how? The CSV is generated from scratch. We don't have a "base" and continue form there. BTW, the same is true for the crd. (and we need to double-check the weird cp of the prev crd when generating the new one)

The only way would be if there were some kubebuilder tags for conversion webhooks, but I checked and there is nothing for it.

I do have a way :) , by using kustomize and a kustomization patch, but this
will involve some copying around and more weird csv-generate stuff that I dislike.
This generation seems like an acceptable trade-off FOR NOW.
Once we take the csv-generator off the table we can find something better.

As always, I am open to suggestions.

Reply:
ok, let's keep it like this for now.